docs/changes/0.22.0.rst

```diff
@@ -0,0 +1,49 @@
+Changelog for QCoDeS 0.22.0
+===========================
+
+The January 2021 release of QCoDeS.
+
+-----------------
+Breaking Changes:
+-----------------
+
+- Removed deprecated SD_common modules from Keysight (#2544)
+- Removed add_parameter from data_set (#2582)
+
+-------------
+Deprecations:
+-------------
+
+- Unused semi private functions modify_values and modify_many_values are deprecated (#2582)
+
+----
+New:
+----
+
+- guid_helpers: Extracting guids from database files and list of strings (#2541)
+
+---------
+Improved:
+---------
+
+- Removed duplicated attributes in channel.py (#2560)
+- Fix header levels and section structure in Measurement example notebook (#2624)
+- doNd notebook update: the notebook compares doNd with Measurement context manager (#2512)
+
+-----------------
+Improved drivers:
+-----------------
+
+- Add checks for signal current level and voltage level queries in Keysight E4980A (#2561)
+- Set signal mode and snapshot_exclude in Keysight E4980A (#2612)
+- The test for awg70000A throws warning when waveform is clipped (#2585)
```

Comment:
Add test to ensure that awg70000A driver throws a warning when waveform is clipped